ihmc-perception/src/main/java/us/ihmc/perception/realsense/RealsenseDeviceManager.java

```diff
@@ -22,6 +21,8 @@
  */
 public class RealsenseDeviceManager
 {
+   public static final String[] REALSENSE_SERIAL_NUMBERS = {"213522252883", "215122253249"};
```

Comment:
Perhaps there's a better place for this list? Also, let me know if you know any other realsense serial numbers. These two are the numbers for the robot, and the one at my desk.

Reply:
One on Nehar's desk: 215122254074

Reply:
However, can't we just remove the serial numbers entirely and just connect to the first one connects?

Reply:
Oh yeah, we absolutely can